### DISCUSSION TITLE
TAC conflit jquery

### DISCUSSION THREAD

**Discussion by Rigolus** [AmauriC/tarteaucitron.js#976] [General]
Bonjour,
Pas très au point sur TAC ni Wordpress ... J'ai besoin de vos lumières.
Le plugin TAC est installé sur plusieurs sites WP, j'ai rencontré des problèmes de chargement de la popin d'autorisation des cookies.
Pour l'un des sites, c'est réglé, il a fallu appelé le plugin dans le footer après l'appel au jquery.
Pour le second, cette méthode ne fonctionne pas. Cependant, un appel a googlemap est actif, serait-ce une source éventuelle de problème de chargement de la popin?

**Comment by AmauriC** [comment]
Bonjour,
Possible d'avoir l'url du site avec le souci? 

Tarteaucitron n'utilise pas jQuery, le problème ne viens pas de là.

**Comment by Rigolus** [comment]
https://qui-sommes-nous.ccas.fr/nos-missions/

Je ne sais pas si tu pourras vraiment voir ce qu'il se passe. Sans s'authentifier ...
Nous avons tenté un attribut 'async=false' dans les appels jquery et autres urls de fichiers js.
Nous pensons à un pb de déclenchement d'évènements dans le mauvais ordre... Mais vraiment pas sur.
Sur ce site la popin TAC se déclenche a peine puis disparait... laissant le layer gris apparent et bloquant.

**Comment by AmauriC** [comment]
C'est Wprocket le problème. Excluez de la minification/optimisation Javascript:

```
/load.js
tarteaucitron.io
```

**Comment by Rigolus** [comment]
Merci Amaury, en plus j'avais trouvé cette solution sur internet et je ne voyais pas comment faire sur wp rocket... je n'avais pas vu la rubrique concernée, j'avais surement le cerveau embrouillé.
Merci encore.

**Comment by Rigolus** [comment]
Bonjour Amauri,
Cette solution fonctionne pour lun des sites c'est déja ca.
J'ai cette erreur qui apparait aussi:
Uncaught TypeError: $ is not a function

Apparemment une histoire d'alias mal géré. Penses tu qu'il faudrait exclure également "js/jquery/jquery.js" de la minification/optimisation?

merci d'avance.

**Reply by AmauriC** [reply]
Sur WordPress l'alias `$` est remplacé par `jQuery` mais ça n'a aucun rapport avec tarteaucitron, c'est un autre plugin.
